plugins/dac_data_manager.c

```diff
@@ -140,7 +141,7 @@ static bool tx_channels_check_valid_setup(struct dac_buffer *dbuf);
 
 static const gdouble abs_mhz_scale = -1000000.0;
 static const gdouble khz_scale = 1000.0;
-static const char *default_channel_names[8] = {
+static const char *default_channel_names[8] = { // TO DO: Should generate these automatically if number of TX will be variable
```

Comment:
this comment is just patch noise if you're (re)moving this in the next patch

Reply:
This didn't show in the final diff.